### DISCUSSION TITLE
Not able to open bruno desktop app(locally)

### DISCUSSION THREAD

**Discussion by yaten2302** [usebruno/bruno#1428] [Q&A] [answered]
Hey @Its-treason,

While I was trying to run the desktop app locally, I was facing some issues. According to the [Contributing.md file](https://github.com/usebruno/bruno/blob/main/contributing.md), we have to run the next app in 1 terminal and the electron app in the other. I'm able to run the `npm run dev:web` (next app) command, but the other one i.e. `npm run dev:electron` (electron app) is showing the following error.

![image](https://github.com/usebruno/bruno/assets/129659514/c03e4848-a91e-4d64-a641-2ee67d12b06a)
<br>
Can anyone please guide me how to resolve this issue?

**Comment by Its-treason** [comment, marked as answer]
The error suggests that `bruno-query` was not build. Did you run `npm run build:bruno-query` to build it?

**Reply by yaten2302** [reply]
Earlier, I didn't run the first 2 commands, now it's working 👍
Thanks
